flask_saml_sso/sso.py

```diff
@@ -244,13 +315,24 @@ def sls(auth):
     Consumes LogoutResponse from IdP when logout has been performed, and
     sends user back to landing page
     """
+    logger = _get_logger()
+    logger.debug('SLS called')
+
     url = auth.process_slo(delete_session_cb=lambda: flask.session.clear())
+    logger.debug('SLS Response XML: \n{}'.format(
+        auth.get_last_response_xml()))
+
     errors = auth.get_errors()
     if errors:
+        logger.error('Errors: {}'.format(errors))
         return _build_error_response(errors)
     if url is not None:
-        return flask.redirect(url)
-    return flask.redirect('/')
+        redirect_to = url
+    else:
+        redirect_to = '/'
+
+    logger.info('Redirecting to {}'.format(redirect_to))
```

Comment:
Perhaps explain _why_ we're redirecting? E.g. “Redirecting to {} for login”?

Reply:
Redirecting is a pretty core part of SSO/SLO as the whole flow is web based, so I thought the _why_ of it was self explanatory.

Reply:
> Redirecting is a pretty core part of SSO/SLO as the whole flow is web based, so I thought the _why_ of it was self explanatory.

You're right, but I figured the logs targeted someone who in operations who might not be intimately familiar with it?

Reply:
I'll add some descriptions.